tests/testthat/test_KIL_pipeline.R

```diff
@@ -0,0 +1,225 @@
+context("Run data quality check on Kilingi Nomme, Estonia, pipeline output")
+
+test_that("KIL outputs all files...", {
+  expect_true("KIL" %in% pipeline_output$Brood_data$PopID)
+  expect_true("KIL" %in% pipeline_output$Capture_data$CapturePopID)
+  expect_true("KIL" %in% pipeline_output$Individual_data$PopID)
+  expect_true("KIL" %in% pipeline_output$Location_data$PopID)
+})
+
+
+test_that("Individual data returns an expected outcome...", {
+
+  #Take a subset of only KIL data
+  KIL_data <- dplyr::filter(pipeline_output$Individual_data, PopID == "KIL")
+
+  #Test 1: Adult tit female
+  #Individual 1742516 should be listed as a female
+  expect_equal(subset(KIL_data, IndvID == "1742516")$Sex_calculated, "F")
+  expect_equal(subset(KIL_data, IndvID == "1742516")$Species, "PARMAJ")
+  expect_equal(subset(KIL_data, IndvID == "1742516")$BroodIDLaid, NA_character_)
+  expect_equal(subset(KIL_data, IndvID == "1742516")$BroodIDFledged, NA_character_)
+  expect_equal(subset(KIL_data, IndvID == "1742516")$RingSeason, 1999)
```

Comment:
Good practice to specify that this is an integer, that way we'll also detect cases where the class of the object is wrong.

```suggestion
  expect_equal(subset(KIL_data, IndvID == "1742516")$RingSeason, 1999L)
```